L1Trigger/DTTriggerPhase2/src/ShowerBuilder.cc

```diff
@@ -0,0 +1,292 @@
+#include "L1Trigger/DTTriggerPhase2/interface/ShowerCandidate.h"
+#include "L1Trigger/DTTriggerPhase2/interface/ShowerBuilder.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+using namespace edm;
```

Comment:
Is it possible to avoid the `using namespace edm` please?

Reply:
Done, thanks!